pack/parallel/rop.json

```diff
@@ -0,0 +1,74 @@
+    "code": "90062",
+    "deck_limit": 1,
+    "deck_options": [
+        { "faction": ["rogue"], "level": { "min": 0, "max": 3 } },
+        { "trait": ["charm", "relic"], "level": { "min": 0, "max": 4 }},
+        { "faction": ["neutral"], "level": { "min": 0, "max": 5 } },
+        {
+            "faction": ["seeker"],
+            "level": { "min": 0, "max": 0 },
+            "limit": 5,
+            "error": "You cannot have more than 5 level 0 Seeker cards"
+        }
+    ],
+    "deck_requirements": "size:30, card:08008:90063, card:08009:90064, random:subtype:basicweakness",
+    "double_sided": true,
+    "faction_code": "rogue",
+    "flavor": "“You call this archaeology?”",
+    "health": 8,
+    "illustrator": "Tony Foti",
+    "is_unique": true,
+    "name": "Monterey Jack",
+    "pack_code": "rop",
+    "position": 62,
+    "quantity": 1,
+    "sanity": 6,
+    "skill_agility": 5,
+    "skill_intellect": 4,
+    "skill_combat": 2,
+    "skill_willpower": 1,
+    "subname": "The Archaeologist",
+    "text": "[reaction] After you discover the last clue at your location: Search the top X cards of your deck for a [[Charm]] or [[Relic]] asset and play it, reducing its cost by X. X is the printed shroud of your location. Shuffle your deck. (Limit once per round.)\n[elder_sign] effect: +1. Gain 1 resource for each [[Charm]] asset and each [[Relic]] asset you control.",
+    "traits": "Wayfarer.",
+    "type_code": "investigator"
+  },
+  {
+    "code": "90063",
+    "cost": 1,
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "illustrator": "Duallbrush",
+    "name": "Trusty Bullwhip",
+    "pack_code": "rop",
+    "position": 63,
+    "quantity": 1,
+    "restrictions": "investigator:08007",
+    "skill_agility": 2,
+    "skill_intellect": 1,
+    "skill_wild": 1,
+    "slot": "Hand",
+    "text": "Fast. Monterey Jack deck only. Advanced.\n[action]: <b>Fight.</b> This attack uses [agility] instead of [combat]. You get +2 skill value for this attack. If this attack succeeds, you may exhaust Trusty Bullwhip to deal +1 damage for this attack and automatically evade the attacked enemy.",
+    "traits": "Item. Weapon. Melee.",
+    "type_code": "asset"
+  },
+  {
+    "code": "90064",
+    "deck_limit": 1,
+    "faction_code": "neutral",
+    "illustrator": "Raymond Bonilla",
+    "name": "Buried Secrets",
+    "pack_code": "rop",
+    "position": 64,
+    "quantity": 1,
+    "restrictions": "investigator:08007",
+    "subtype_code": "weakness",
+    "text": "Advanced.\n<b>Revelation</b> - Put Buried Secrets into play in your threat area.\nIf your location can be investigated, you cannot move except by scenario card effects.\n[action][action]: <b>Investigate.</b> If you succeed, instead of discovering clues, discard Buried Secrets.",
```

Comment:
```suggestion
    "text": "Advanced.\n<b>Revelation</b> - Put Buried Secrets into play in your threat area.\nIf your location can be investigated, you cannot move except by scenario card effects.\n[action] [action]: <b>Investigate.</b> If you succeed, instead of discovering clues, discard Buried Secrets.",
```